keadm/cmd/keadm/app/cmd/ctl/ctl.go

```diff
@@ -0,0 +1,26 @@
+package ctl
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/ctl/get"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/ctl/restart"
+)
+
+var (
+	ctlLongDescription  = `"keadm ctl" command operates on the data surface of edge nodes`
+	ctlShortDescription = `ctl function to operate on the data surface of edge nodes`
```

Comment:
```suggestion
	ctlShortDescription = `Commands operating on the data plane at edge`
```
We can use the same for Long and Short?